webplugin/js/app/kommunicate.js

```diff
@@ -116,6 +116,47 @@ $applozic.extend(true, Kommunicate, {
             }, SET_TIMEOUT_DURATION);
         }
     },
+    updateConversationMetadata: function (conversationMetadata) {
+        if (
+            conversationMetadata != null &&
+            typeof conversationMetadata !== 'undefined'
+        ) {
+            var metadataToSend = Object.assign({}, conversationMetadata);
+            if (
+                kommunicateCommons.isObject(metadataToSend) &&
+                kommunicateCommons.isObject(metadataToSend.metadata) &&
+                metadataToSend.groupId &&
+                metadataToSend.metadata
+            ) {
+                var metadataToShow = JSON.stringify(metadataToSend.metadata);
+                const groupDataResponse = Applozic.ALApiService.groupUpdate({
+                    data: {
+                        groupId: metadataToSend.groupId,
+                        metadata: {
+                            metadataToShow: metadataToShow,
```

Comment:
shall we rename it to conversationMetadata?
then it will be consistent naming everywhere docs, code, and dashboard